cocaine/tools/actions/keyring.py

```diff
@@ -27,6 +31,9 @@ def execute(self):
 
 class Update(Action):
     def __init__(self, cid, tvm):
+        if tvm.ty() != TVM.ty():
```

Comment:
Why isn't it checked via `isinstance(tvm, TVM)`?

Reply:
Reverted, because actions with keyring use "tvm" service directly.